src/main/java/Kevin.java

```diff
@@ -0,0 +1,150 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class Kevin{
+
+    private static final List<Task> taskList = new ArrayList<>();
+    private static final String line = "____________________________________________________________";
+    public static void main(String[] args) {
+        Kevin.printGreetMessage();
+        Scanner scanner = new Scanner(System.in);
+        String userInput = scanner.nextLine();
+        label:
+        while (true) {
+            // Prints out the bye message
+            String[] splitMessage = userInput.split(" ");
+            String instruction = splitMessage[0];
+            switch (instruction) {
+                case "bye":
+                    Kevin.printByeMessage();
+                    break label;
+                case "list":
+                    // Prints out the list
+                    Kevin.listAllTasks();
+                    break;
+                case "mark": {
+                    Kevin.markTaskDone(splitMessage[1]);
+                    break;
+                }
+                case "unmark": {
+                    Kevin.unmarkTaskDone(splitMessage[1]);
+                    break;
+                }
+                case "delete": {
+                    try {
+                        Kevin.deleteTask(splitMessage[1]);
+                    } catch (TaskListEmptyException e) {
+                        System.out.println(line);
+                        System.out.println(e.getMessage());
+                        System.out.println(line);
+                    }
+                    break;
+                }
+                default:
+                    // Add new tasks to the task list
+                    try {
+                        Kevin.addNewTask(userInput);
+                    } catch (DescriptionIncompleteException e1) {
+                        System.out.println(line);
+                        System.out.println(e1.getMessage());
+                        System.out.println(line);
+                    } catch (IllegalCommandException e2) {
+                        System.out.println(line);
+                        System.out.println(e2.getMessage());
+                        System.out.println(line);
+                    }
+            }
```

Comment:
I think there is no indentation for case clauses and no need curly braces, example as below 😃 
```
switch (condition) {
case ABC:
    statements;
    // Fallthrough
case DEF:
    statements;
    break;
case XYZ:
    statements;
    break;
default:
    statements;
    break;
}
```